nautobot/core/templates/inc/nav_favorites.html

```diff
@@ -0,0 +1,31 @@
+<ul class="list-unstyled sidenav-favorite-list">
+    {% for item in request.user.navbar_favorites %}
+        {% if item.link and item.name %}
+            <li class="sidenav-link-wrapper" id="sidenav-favorite-link-{{ forloop.counter }}">
+                <a class="sidenav-link" href="{{ item.link }}">
+                    <span class="sidenav-link-text">{{ item.name }}</span>
+                    {% if item.tab_name and item.group_name %}
+                    <span class="sidenav-link-context">{{ item.tab_name }} / {{ item.group_name }}</span>
+                    {% endif %}
+                </a>
```

Comment:
less custom css and custom classes = less noise. prefer proper html elements and bootstrap utils over custom styles when possible
```suggestion
                    <span>
                        {{ item.name }}
                        {% if item.tab_name and item.group_name %}
                            <br />
                            <small class="text-secondary">{{ item.tab_name }} / {{ item.group_name }}</small>
                        {% endif %}
                    </span>
```

Reply:
Not sure why but I wanted to avoid `<br />`. Maybe `vstack` will be better? What do you think? 

In your suggestion even the `<span>` is not needed, right?